src/R/Components/Impl/Controller/IMenuCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.R.Components.Controller {
+    /// <summary>
+    /// An object that implements a single command. 
+    /// </summary>
+    public interface IMenuCommand {
```

Comment:
Agreed

---
In reply to: [63588679](https://github.com/Microsoft/RTVS/pull/1785#discussion_r63588679) [](ancestors = 63588679)